feature-pack/src/main/resources/content/bin/appclient.conf

```diff
@@ -42,7 +42,7 @@ fi
 # Specify options to pass to the Java VM.
 #
 if [ "x$JAVA_OPTS" = "x" ]; then
-   JAVA_OPTS="-Xms64m -Xmx512m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
```

Comment:
How sure are you that there is no 8.x JVM that still uses permgen?  Remember we don't just run on Oracle JVM.